### ISSUE TITLE
Documentation issue -> api.send_invoice() example

### ISSUE THREAD

**Issue by roboyeti** [minimul/qbo_api#130]
RE: api.send_invoice(invoice_id: 1, email_address: 'billy@joe.com') example
Problem:
  1.> api doesn't match the rest of the examples, probably should be: qbo_api
  2.> There is no send_invoice() call, that I or Ruby could find.  Looking thru the code, it appears to be .deliver()
  3.> Parameters appear to be different as well for solution, see below.
  
Solution (not documented)
  The following works for me, following along with the doc example from the start:
   qbo_api.deliver(:invoice, { entity_id: <invoice_id>, params: [ email_address: <email_address> })

f someone is still active enough around here to merge changes, I can certainly make the doc changes and add a send_invoice, since that would make sense to have as a common action.

**Comment by minimul** [comment]
Yup, it is `.deliver` ref: https://github.com/minimul/qbo_api/pull/112/files

Nice catch.

I'll merge the documentation changes.
